#sanwa 400-JYP62US controller axis numbers and buttun numbers in the joy topics.
---
#velorosity
speedAxis     : 1
turnAxis      : 0
#maxSpeed      : 0.5
#maxTurnW      : 1.570796327
#flipper.fr
frUpButton    : 8
frDownButton  : 9
#frW           : 0.314
#frDefaultAngle: 0.785398163
#flipper.fl
flUpButton    : 6
flDownButton  : 7
#flW           : 0.314
#flDefaultAngle: 0.785398163
#flipper.rr
rrUpButton    : 14
rrDownButton  : 15
#rrW           : 0.314
#rrDefaultAngle: 0.785398163
#flipper.rl
rlUpButton    : 12
rlDownButton  : 13
#rlW           : 0.314
#rlDefaultAngle: 0.785398163
...
